homeassistant/components/dpi4relay/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "dpi4relay",
+  "name": "DockerPi 4 Channel Relay expansion board",
+  "documentation": "https://wiki.52pi.com/index.php/DockerPi_4_Channel_Relay_SKU:_EP-0099",
```

Comment:
The documentation link should be "https://www.home-assistant.io/components/<domain>".
See the [developer documentation](https://developers.home-assistant.io/docs/en/creating_integration_manifest.html) for more details about the manifest.json file.